docs/docs/getting-started/deployment.md

```diff
@@ -290,10 +353,19 @@ If you don't know the correct values use `{"BootloaderVersion":255,"TEEVersion":
 contrast generate --reference-values metal-qemu-tdx resources/
 ```
 
-:::note[Missing TCB values]
-On bare-metal TDX, `contrast generate` is unable to fill in the `MinimumTeeTcbSvn` and `MrSeam` TCB values of the created manifest.json as they can vary between platforms.
-They will have to be filled in manually.
-If you don't know the correct values use `ffffffffffffffffffffffffffffffff` and `000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000` respectively and observe the accurate ones in the error messages in the following steps. This should only be done in a secure environment.
+On bare-metal TDX, `contrast generate` is unable to fill in the `MrSeam` value as it depends on your platform configuration.
+It will have to be filled in manually.
```

Comment:
`MinimumTeeTcbSvn` needs to be filled in, too, right?

Reply:
Not since https://github.com/edgelesssys/contrast/commit/bc0ebf82ff6d24c656a7fbb84f3cd16c9b967dd2